src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts

```diff
@@ -121,10 +121,9 @@ function useSidebarMetadataFetcher(
 
     const fetchFileSuccessCallback = React.useCallback(
         (fetchedFile: BoxItem) => {
-            const { currentFile } = file ?? {};
-            const currentFileCanUpload = getProp(currentFile, FIELD_PERMISSIONS_CAN_UPLOAD, false);
+            const currentFileCanUpload = getProp(file, FIELD_PERMISSIONS_CAN_UPLOAD, false);
```

Comment:
note for box: this change doesn't seem correct. the logic between the original line and this line change is different